NetPyNE.js

```diff
@@ -134,10 +134,13 @@ export default class NetPyNE extends React.Component {
 			return {
 				value: tab,
 				prevValue: tab, 
-				freezeInstance: args.freezeInstance!=undefined?args.freezeInstance:fi,
-				freezeSimulation: args.freezeSimulation!=undefined?args.freezeSimulation:fs,
+				freezeInstance: args.freezeInstance != undefined ? args.freezeInstance : fi,
+				freezeSimulation: args.freezeSimulation != undefined ? args.freezeSimulation : fs,
 			}
 		});
+		if (tab=="define") {
```

Comment:
Is this new? what is this for? can we move it to another PR if it is not strictly required?

Reply:
that is to hide the widgets. we don't want all the plots in the define tab